frontend/src/components/HibernationSchedule.vue

```diff
@@ -128,11 +128,19 @@ export default {
     reset () {
       this.parseSchedules(this.scheduleCrontab)
     },
+    scheduleEventObjFromRegex (regex) {
```

Comment:
The name of the parameter should be something like `crontabLine`. I think the `console.warn` should be done in the `scheduleEventObjFromRegex` function if the entry exists but could not be parsed.